compose/snippets/src/main/java/com/example/compose/snippets/components/TimePickers.kt

```diff
@@ -153,6 +181,57 @@ fun InputExample(
 }
 // [END android_compose_components_input]
 
+@OptIn(ExperimentalMaterial3Api::class)
+// [START android_compose_components_advanced]
+@Composable
+fun AdvancedExample(
```

Comment:
Can you name this `AdvancedTimePickerExample()`

Reply:
Done!